### PR TITLE
[improvement] Improve Add multiple users UX

### PR DESCRIPTION
**Pull request by JayaShakthi97**
Improved UX
- Refer: https://github.com/wso2/product-is/issues/17670#issuecomment-1809448106

image

image

https://github.com/wso2/identity-apps/assets/37938529/d0f7ddaa-1cd6-4d4f-bc29-57935c4043e1

### Related Issues
- https://github.com/wso2/product-is/issues/17670

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/twelve-coins-design.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/myaccount": patch
+"@wso2is/common": patch
```
Comment (JayaShakthi97) [inline .changeset/twelve-coins-design.md:3]: i18n module is used in common module as well.

**Review by Achintha444 — DISMISSED** [review]

File: .changeset/twelve-coins-design.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/myaccount": patch
```
Comment (dasuni-30) [inline .changeset/twelve-coins-design.md:2]: Can we patch the console as well here?

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 3]
```diff
@@ -1912,6 +1927,18 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
 
             <Modal.Content className="content-container" scrolling>
                 <Grid>
+                    <>
+                        <Grid.Row columns={ 1 }>
+                            <Grid.Column>
+                                <Alert severity="info">
+                                    {
```
Comment (dasuni-30) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:1934]: We need to provide the same experience that we give in single user creation.

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 4]
```diff
@@ -11151,7 +11152,7 @@ export const console: ConsoleNS = {
                             alerts: {
                                 importSuccess: {
                                     description: "The user accounts were imported successfully.",
-                                    message: "Import Successful"
+                                    message: "User Creation Successful"
```
Comment (dasuni-30) [inline modules/i18n/src/translations/en-US/portals/console.ts:11155]: Updating the alert here affects both manual and file based approaches.

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**